### ISSUE TITLE
LMSYS-Chat-1M Performance Inconsistency

### ISSUE THREAD

**Issue by clearloveclearlove** [WhoTHU/detection_logits#1]
Hi, Zhanhao!

Thanks for your interesting work and code! The results I obtained from running your code are shown in the table below.

|      MULI      | AUPRC | TPR@FPR10% | TPR@FPR1% | TPR@FPR0.1% | TPR@FPR0.01% |
|:--------------:|:-----:|:----------:|:---------:|:-----------:|:------------:|
|    ToxicChat   | 90.13 |    98.62   |   80.39   |    39.78    |     21.55    |
|  LMSYS-Chat-1M | 74.03 |    92.42   |   68.69   |    21.21    |     6.57     |

I achieved results on the ToxicChat dataset that are similar to those reported in the paper, but the results on the LMSYS-Chat-1M dataset show a significant decline. Do you know the reason for this, and do you have any suggestions on how to reproduce the results from your paper?

**Comment by WhoTHU** [comment]
Hi! May I ask what labels did you use for LMSYS-Chat-1M?

**Comment by clearloveclearlove** [comment]
> Hi! May I ask what labels did you use for LMSYS-Chat-1M?

Hi, I didn't modify any code; I just ran the code you released.

#
split_name = 'train' # train, test
content = datas_tt[split_name]['content']
y_toxicity = datas_tt[split_name]['toxicity_label']
results_tt = datas_tt[split_name]['logits']

**Comment by WhoTHU** [comment]
Oh, I found the reason; I uploaded a wrong version for LMSYS-Chat-1M, which used wrong labels. I'll update it soon

**Comment by WhoTHU** [comment]
Fixed!
